core/chains/evm/logpoller/parser.go

```diff
@@ -60,19 +59,17 @@ func (v *pgDSLParser) Block(p primitives.Block) {
 	)
 }
 
-func (v *pgDSLParser) Confirmations(p primitives.Confirmations) {
-	switch p.ConfirmationLevel {
-	case primitives.Finalized:
+func (v *pgDSLParser) Confidence(p primitives.Confidence) {
```

Comment:
Shouldn't this be handled in the same way as `Comparator` since it is also always getting remapped to its EVM specific counterpart `confirmationsFilter` ?

Reply:
It is remapped for EVM from `Confidence` as a general handler to `VisitConfirmationsFilter` where specific EVM confidence -> confirmations mappings are configured.

Reply:
Look at `remapPrimitive` in `event_binding.go`